packages/studio-base/src/i18n/zh/panelSettings.ts

```diff
@@ -0,0 +1,122 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+export default {
+  importOrExportSettings: "导入/导出设置",
+  panelSettings: "面板设置",
+  noLayoutSelected: "<selectLayoutLink>选择一个布局</selectLayoutLink> 开始使用！",
+  selectAPanelToEditItsSettings: "选择一个面板以编辑其设置。",
+  loadingPanelSettings: "正在加载面板设置...",
+  currentSettingsPanelName: "{{title}}面板设置",
+  panelName: "{{title}}面板",
+  panelNoAnySettings: "{{panelTitle}}面板没有任何设置",
+  panelTitle: "{{title}}",
+  this: "当前",
+  resetToDefault: "重置为默认值",
+  searchPanelSettings: "搜索面板设置...",
+  clearSearch: "清空搜索",
+  frame: "参考系",
+  displayFrame: "显示坐标系",
+  displayFrameHelp: "放置相机用于显示坐标系。相机的位置和方向将相对于该坐标系的原点。",
+  followMode: "跟踪模式",
+  followModeHelp: "更改回放期间相机的行为，是否跟踪显示坐标系。",
+  scene: "场景",
+  reset: "重置",
+  renderStats: "渲染统计",
+  background: "背景",
+  labelScale: "标签比例",
+  labelScaleHelp: "应用于所有标签的比例因子",
+  ignoreColladaUpAxis: "忽略COLLADA的<up_axis>",
+  ignoreColladaUpAxisHelp: "通过忽略COLLADA文件中的<up_axis>标记，匹配rviz的行为",
+  takeEffectAfterReboot: "此设置需要重新启动以生效",
+  syncCamera: "同步相机",
+  syncCameraHelp: "将相机与其他启用此设置的面板同步。",
+  meshUpAxis: "网格上轴",
+  meshUpAxisHelp: "加载缺少方向信息的网格（STL和OBJ）时使用的“上”方向",
+  YUp: "Y-up",
+  ZUp: "Z-up",
```

Comment:
Is it normal to say "up" in english?
```suggestion
  YUp: "Y上",
  ZUp: "Z上",
```

Reply:
> Is it normal to say "up" in english?

In fact, for very simple English or nouns, the Chinese in general will also use the English name, and here the meaning is actually difficult to be accurately translated into Chinese, in Chinese '上' is a lot of meaning, so I think that rather than ambiguous forced translation, it is easier to use English directly to make people understand, in fact, many agreed vocabulary even in China we will also use English